tasks/interactive-bots.md

```diff
@@ -41,7 +41,7 @@ Follow the tutorial below to test run the followup bot on your Zulip instance
 
 * Run the Zulip server and log in the browser as Cordelia.
 
-* Find the API keys.
+* Find the API keys (Settings -> Your bots -> Show/change your API).
```

Comment:
I wanted to approve my review, but the brackets with the text are still on line 44.

@abhayKashyap03 could you take a look?